bidwire/bid.py

```diff
@@ -17,36 +17,36 @@ class Bid(Base):
     location = Column(Text)
     open_date = Column(DateTime)
     items = Column(JSON)
+    site = Column(Text)
 
     def __repr__(self):
         return "<Bid(id={}, identifier={}, description={}, created_at={})>".format(
             self.id, self.identifier, self.description, self.created_at)
 
-    def url(self):
```

Comment:
I feel that this `url` method should definitely be a method on the Bid, not on the notifier. Let's bring it back here. You can use the enum to switch the logic of how to construct the URL.